package.json

```diff
@@ -25,10 +25,18 @@
     "es6",
     "babel"
   ],
-  "scripts":{
+  "scripts": {
+    "lint": "eslint .",
```

Comment:
We should probably delegate the linting to a script on the package, but I agree that we should have a script at the root level, and that the config should be controlled at the root level.

At the minimum though, we should only run lint on the `packages/` folder and make sure to skip `node_modules` folders, and bundles.

```
eslint packages --ignore-pattern node_modules --ignore-pattern bundle.js
```

Reply:
You're right I will add an `.eslintignore` file to specify bundle.js

But node_modules are ignored by default, see https://eslint.org/docs/user-guide/configuring#eslintignore
> In addition to any patterns in a .eslintignore file, ESLint ignores files in /**/node_modules/* by default. It can still be added using !.

Reply:
I have checked with `eslint . --debug` and I can confirm that node_modules folders were not parsed

Reply:
I added an .eslintignore file that allows the `lint` command to work even after building the `react-vis` package.

The showcase's build command fails on this branch so we should add the needed ignored files once the build is fixed

Reply:
nicely done